src/components/voting/VoterButtons.tsx

```diff
@@ -99,14 +100,15 @@ const VoterButton = React.memo(
     }
 
     let icon: JSX.Element
-    const label = preview || isMobile ? undefined : newKind
+    const labelText = isUpvote ? 'Like' : 'Dislike'
```

Comment:
Remove label

Reply:
I refactored, so I remove the preview props, as we no longer use any actions with label
but the label itself I still keep it as the button title so the button can have built in tooltip
![image](https://github.com/dappforce/polkaverse/assets/53143942/d9372c0e-de0f-4168-88e1-70f765a2e2a0)